bitnami/kube-prometheus/README.md

```diff
@@ -476,12 +478,26 @@ This chart includes a `values-production.yaml` file where you can find some para
 
 ### Additional scrape configurations
```

Comment:
As this is a breaking change, please, add an entry in the notable changes section, try to show a path for the upgrade.